### PR TITLE
fix : better pk creation for civility

### PR DESCRIPTION
**Pull request by FHenry**
# Instructions
*This is a template to help you make good pull requests. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *only keep the "Fix", "Close" or "New" section*
- *follow the project [contributing guidelines](/.github/CONTRIBUTING.md)*
- *replace the bracket enclosed textswith meaningful informations*

# Fix #[*issue_number Short description*]
[*Long description*]

# Close #[*issue_number Short description*]
[*Long description*]

# New [*Short description*]
[*Long description*]

### CODE REVIEW

File: htdocs/install/mysql/tables/llx_c_civility.sql [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 create table llx_c_civility
 (
-  rowid       integer    PRIMARY KEY,
+  rowid       integer AUTO_INCREMENT PRIMARY KEY NOT NULL
```
Comment (eldy) [inline htdocs/install/mysql/tables/llx_c_civility.sql:22]: If we do so, we should also change into migration script. According to sample we should add this into migration script

-- To make pk to be auto increment (mysql):    
--VMYSQL4.3 ALTER TABLE llx_c_civility CHANGE COLUMN rowid rowid INTEGER NOT NULL AUTO_INCREMENT;

-- To make pk to be auto increment (postgres):
--VPGSQL8.2 CREATE SEQUENCE llx_c_civility OWNED BY llx_c_civility.rowid;
--VPGSQL8.2 ALTER TABLE llx_c_civility ADD PRIMARY KEY (rowid);
--VPGSQL8.2 ALTER TABLE llx_c_civility ALTER COLUMN rowid SET DEFAULT nextval('llx_c_civility_rowid_seq');
--VPGSQL8.2 SELECT setval('llx_c_civility_rowid_seq', MAX(rowid)) FROM llx_c_civility;

Comment (eldy) [reply htdocs/install/mysql/tables/llx_c_civility.sql:22]: I did not test on pgsql however

**Comment by eldy** [comment]
Oops 256 files modified. somethinig seems wrong with PR

File: htdocs/install/mysql/tables/llx_c_civility.sql (hunk 1 above)
Comment (FHenry) [reply htdocs/install/mysql/tables/llx_c_civility.sql:22]: Tested on pgsql with this, it's ok

--VPGSQL8.2 CREATE SEQUENCE llx_c_civility_rowid_seq OWNED BY llx_c_civility.rowid;
--VPGSQL8.2 ALTER TABLE llx_c_civility ALTER COLUMN rowid SET DEFAULT nextval('llx_c_civility_rowid_seq');
--VPGSQL8.2 SELECT setval('llx_c_civility_rowid_seq', MAX(rowid)) FROM llx_c_civility;

**Pull request merged**